welcome/revhistory_full.adoc

```diff
@@ -4,6 +4,15 @@
 :data-uri:
 :icons:
 :experimental:
+== Wed Apr 27 2016
+.Admin Guide
+include::admin_guide/revhistory_admin_guide.adoc[tag=admin_guide_wed_apr_27_2016]
+
+.Architecture
+include::architecture/revhistory_architecture.adoc[tag=architecture_wed_apr_27_2016]
+
+.Install Config
+include::install_config/revhistory_install_config.adoc[tag=install_config_wed_apr_27_2016]
```

Comment:
@bfallonf Can you rename these "Admin Guide" and "Install Config" table titles? This is a current limitation of the release script. See how the full names are spelled out here: https://docs.openshift.com/enterprise/latest/welcome/revhistory_full.html

Also the script dumps this stuff above the intro sentence for this topic, so you have to cut/paste it back in place to fix it. See the current line 17 here below.